7_react/react1/week1/src/Header/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import "./Header.css"
+function Header(){
+    return(
+    <>
```

Comment:
You already have a wrapper HTML component here, so we can just use that on the top level:
```
return(
    <header className='header'>
      <h1 className='headerTitle'>TODO LIST</h1>
    </header>
    )
```

Reply:
Also it is nice to pay attention to indentation, so the code becomes more readable:
```html
<parent1> {/* level 1*/}
  <nodeWithoutChildren /> {/* level 2*/}
  <nodeWithChildren>
    <nodeWithAnotherChildren> {/* level 3*/}
      <h1> Hello :) </h1> {/* level 4*/}
    </nodeWithAnotherChildren>
    <p> This is a description </p>
  </ nodeWithChildren>
</parent1>
```